mesh/mesh.hpp

```diff
@@ -431,7 +431,7 @@ class Mesh
 
    /// Swaps internal data with another mesh. By default, non-geometry members
    /// like 'ncmesh' and 'NURBSExt' are only swapped when 'non_geometry' is set.
-   void Swap(Mesh& other, bool non_geometry = false);
+   void Swap(Mesh& other, bool non_geometry);
```

Comment:
Is it necessary to remove the default parameter?

Reply:
I was having second thoughts about the default. Defaults can be dangerous and it's better to be explicit. I don't insist on removing it though.

Reply:
It is a protected method, so it is not a big deal to change it (i.e. remove the default value).